### ISSUE TITLE
Add support for color scheme configuration

### ISSUE THREAD

**Issue by trincasidra** [mk-5/fjira#93]
**Describe the solution you'd like**

I'd like to be able to configure the color scheme, for example changing a file such as `~/.config/fjira/fjira.conf`.

**Describe alternatives you've considered**

- Another option would be for fjira to inherit the color scheme of the terminal it's running in... but that looks harder to do.
- I also checked if I could customize tcell colors globally, but I don't think that's possible, as all color names are hardcoded in colors.go.

**Comment by mk-5** [comment]
@trincasidra color scheme sounds good :)

do you have any proposal about the schema itself? what(which parts) should be "configurable"?

**Comment by trincasidra** [comment]
Nice! Thanks :)

Personally the ones I'd really love to change the most are:
- App background and foreground.
- Bottom bar background and foreground. For the shortcut text, `BottomBarActionBarKeyBold`, maybe it would make sense to have the option to use the app background color (or just default to that if not provided).
- Top bar background and foreground.
- Kanban board headers background and foreground.
- Highlighted issue background and foreground (for both list and board views).

But in the end, I think being able to change everything would be nice to have:
- Issue details' boxes title and border. I believe it would be super nice to be able to have a separate color for the box title and box border, but I'm unsure if it's technically possible.
- The highlighted issue's red `>` symbol on the left-most side.
- The yellow input help text (ie. "select issue...") and project/issue counter.
- The green and underlined highlighted characters when searching/filtering.
- The red foreground when selecting an issue in a board.
- I would also really appreciate to be able to set up some conditional coloring on the status column (ie. like making "DONE" green, "CANCELLED" gray or "BLOCKED" red).
- Similarly, the assignee column could have something similar (thinking gray for "Unassigned")

**Comment by mk-5** [comment]
@trincasidra I totally agree - it would be really nice feature :) I had a plan to standardise colors within the app a bit anyway - I'll do it 👍 

I think I can take this up with the beginning of the new year  - just follow that issue, I will make a note as soon as feature is ready

**Comment by trincasidra** [comment]
Great! Glad to hear :)

**Comment by mk-5** [comment]
@trincasidra ready in version 1.2.1

Custom color scheme could be defined in `~/.fjira/colors.yml` file. Default color scheme file is here: [colors.yml](https://github.com/mk-5/fjira/blob/master/assets/colors.yml)

Please let me know if it fits :)

**Comment by mk-5** [comment]
@trincasidra interesting - I will check how it goes on my local

**Comment by mk-5** [comment]
@trincasidra it's "okay" on my local - I'm using `iTerm 2` on MacOS.

![image](https://github.com/mk-5/fjira/assets/31850900/74d1d814-d014-43a1-87f7-a1cf4076857e)

Default background color is `#161616` which is equivalent to 22,22,22

Which terminal are you using?

**Comment by trincasidra** [comment]
I'm using bash over kitty, however I've tried this on some other terminals and I get the same  result:
![image](https://github.com/mk-5/fjira/assets/15145418/900a17e8-a810-42f1-9984-cb13a97f9a1e)

**Comment by mk-5** [comment]
@trincasidra super weird issue 😄  I will try to dig a bit deeper and find something
